lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -1020,3 +1017,17 @@ def generate(cls, _xmodule_instance_args, _entry_id, course_id, task_input, acti
         }
 
         return task_progress.update_task_state(extra_meta=current_step)
+
+    @staticmethod
+    def _generate_upload_file_name(problem_locations, filters):
+        """Generate a concise file name based on the report generation parameters."""
+        multiple_problems = len(problem_locations) > 1
+        csv_name = 'student_state'
+        if multiple_problems:
+            csv_name = '{}_from_multiple_blocks'.format(csv_name)
```

Comment:
Nit (really, feel free to ignore): I personally find it a bit unclear / hard to read code that uses `blah = "{}...".format(blah, ...)` to append to an existing string.

```
csv_name += '_from_multiple_blocks'
```
makes it more clear that it's a simple append.

Reply:
Done.

Reply:
Thanks! Please squash now :)

Reply:
Done.